static/app/views/monitors/components/detailsSidebar.tsx

```diff
@@ -14,31 +14,18 @@ import {getFormattedDate} from 'sentry/utils/dates';
 import {DEFAULT_MAX_RUNTIME} from 'sentry/views/monitors/components/monitorForm';
 import MonitorIcon from 'sentry/views/monitors/components/monitorIcon';
 import {
-  IntervalConfig,
   Monitor,
   MonitorEnvironment,
   MonitorStatus,
   ScheduleType,
 } from 'sentry/views/monitors/types';
+import {scheduleAsText} from 'sentry/views/monitors/utils';
 
 interface Props {
   monitor: Monitor;
   monitorEnv?: MonitorEnvironment;
 }
 
-function getIntervalScheduleText(schedule: IntervalConfig['schedule']) {
```

Comment:
wrote this before, but realized i should just be reusing the code from the monitors listing page